Sources/Client/Model/Event.swift

```diff
@@ -416,58 +416,70 @@ public enum Event: Decodable {
 extension Event: Equatable {
     public static func == (lhs: Event, rhs: Event) -> Bool {
```

Comment:
This custom implementation is dangerous and I think we should make it safer somehow. I see two options:
- add tests for all events that will check the equality works as expected,
- make all associated types used in this `enum` equatable a let the compiler generate this for us.

Reply:
I thought you would do either a) or b), not both :) But it doesn't hurt to have the tests in place :)

Reply:
I did a), then found that I can remove the custom implementation and it was a shame to delete.